src/overlays/actors/ovl_En_Ru2/z_en_ru2.c

```diff
@@ -111,41 +140,41 @@ void EnRu2_Destroy(Actor* thisx, PlayState* play) {
     Collider_DestroyCylinder(play, &this->collider);
 }
 
-void func_80AF2608(EnRu2* this) {
+void EnRu2_UpdateEyeTextures(EnRu2* this) {
     s32 pad[3];
-    s16* unk_2A6 = &this->unk_2A6;
-    s16* unk_2A4 = &this->unk_2A4;
+    s16* eyeTextureFrame = &this->eyeTextureFrame;
+    s16* eyeTextureIndex = &this->eyeTextureIndex;
 
-    if (DECR(*unk_2A6) == 0) {
-        *unk_2A6 = Rand_S16Offset(0x3C, 0x3C);
+    if (DECR(*eyeTextureFrame) == 0) {
+        *eyeTextureFrame = Rand_S16Offset(0x3C, 0x3C);
     }
 
-    *unk_2A4 = *unk_2A6;
-    if (*unk_2A4 >= 3) {
-        *unk_2A4 = 0;
+    *eyeTextureIndex = *eyeTextureFrame;
+    if (*eyeTextureIndex >= 3) {
+        *eyeTextureIndex = 0;
     }
 }
 
-s32 func_80AF2690(EnRu2* this) {
+s32 EnRu2_GetParamsHighByte(EnRu2* this) {
```

Comment:
```suggestion
s32 EnRu2_GetSwitchFlag(EnRu2* this) {
```
also rename `params_shift` in this function to `switchFlag`